src/journal/Journaler.cc

```diff
@@ -444,10 +451,10 @@ void Journaler::get_metadata(uint8_t *order, uint8_t *splay_width,
 std::ostream &operator<<(std::ostream &os,
 			 const Journaler &journaler) {
   os << "[metadata=";
-  if (journaler.m_metadata != NULL) {
+  if (journaler.m_metadata) {
     os << *journaler.m_metadata;
   } else {
-    os << "NULL";
+    os << "failed to find and replace";
```

Comment:
Nit: in my original comment, I was attempting to say no need to change this line

Reply:
@dillaman We should say something here which is more helpful for users, shouldn't we?

Reply:
Sure -- you can say "uninitialized"